application/views/admin/token/bounce.php

```diff
@@ -98,9 +98,9 @@
                                         'name' => 'bounceaccountencryption',
                                         'value'=> $settings['bounceaccountencryption'] ,
```

Comment:
Same here.

Reply:
You are right here. Ideally I would have used database migration script for this, but it looked very complicated to me. Fix adjusted here and in global _bounce.php too.